### PR TITLE
User events index

### PR DESCRIPTION
**Pull request by annekemcgrady**
## Changes proposed in this pull request:
* Create user events index spec
* Create user events controller with route
* Create user events serializer

## Card(s) closed in this pull request:
Does not close any card as still need to write sad path test and document in README

## What did you struggle on to complete?
* Strange error that was resolved with updating a configuration in development environment

### Test Coverage Percentage: 100%
- [x] Tested my new feature(s) as well as any feasible edge cases (if possible)
- [x] Checked coverage/index.html - did not add any new code that's not covered by testing (if possible)
- [x] Ran the test suite - all tests are passing
- [x] Merged in the latest master to my branch with git pull origin master & resolved merge conflicts
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@
   config.cache_classes = false
 
   # Do not eager load code on boot.
-  config.eager_load = false
+  config.eager_load = true
```
Comment (lpile) [inline config/environments/development.rb]: That was an interesting error.
